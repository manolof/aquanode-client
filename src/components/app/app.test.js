import { shallow } from 'enzyme';
import React from 'react';

import App from './app';

describe('App Component', () => {
	afterEach(() => {
		jest.resetAllMocks();
	});

	afterAll(() => {
		jest.restoreAllMocks();
	});

	it('should render', () => {
		const wrapper = shallow(<App/>);

		expect(wrapper.getElements()).toMatchSnapshot();
	});
});
