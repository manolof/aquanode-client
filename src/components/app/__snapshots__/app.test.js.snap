// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render 1`] = `
Array [
  <div
    className="wrapper"
>
    <header>
        <Connect(StatusContainer) />
    </header>
    <div
        className="content"
    >
        <Connect(ScheduleContainer) />
    </div>
</div>,
]
`;
